src/aarch64/cpu-aarch64.cc

```diff
@@ -393,7 +393,6 @@ uint32_t CPU::GetCacheType() {
   // Copy the content of the cache type register to a core register.
   __asm__ __volatile__("mrs %[ctr], ctr_el0"  // NOLINT(runtime/references)
                        : [ctr] "=r"(cache_type_register));
-  VIXL_ASSERT(IsUint32(cache_type_register));
   return static_cast<uint32_t>(cache_type_register);
```

Comment:
Thanks for this PR. The return type of the prototype and the returning point here also need 64 bits data type as well. The  VIXL MTE branch development is on the way. Do you think is it good to wait until MTE support is released?

Reply:
Oops, right I can convert that to a 64-bit.
If the MTE branch is getting merged soon that would be fine. I already have the crash fix in my upstream branch.
It crashes even without using MTE of course. I just wanted to resolve the crash that I had and I assume other people will start running in to.